copter/source/docs/land-mode.rst

```diff
@@ -31,6 +31,12 @@ features:
     second.  It does not use the altitude to decide whether to shut off the
     motors except that the copter must also be below 10m above the home
     altitude.
+    
+.. note::
+    
+    Using a Weight on Wheels (WoW) switch will increase the decent rate and
+    accelerometer ranges that are acceptable for landing detection. This
+    feature is enabled anytime that LGR_WOW_PIN is not disabled.
```

Comment:
anytime the...

Reply:
are there any other relevant parameters needing setup to make this work correctly?

Reply:
LGR_WOW_POL for the polarity of the switch, but this feature is not dependent on that parameter.

Reply:
@TunaLobster Charlie, needs merge conflicts resolved before it can be merged..thanks